components/island/IslandGroup.js

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Box from '../box';
+import theme from './theme.css';
+
```

Comment:
Why pass `color`, but not `dark`? Pass both or don't pass any?

```jsx
const lightColors = ['white', 'neutral'];

isDark () => {
	const { dark, color } = this.props;
	if (lightColors.includes(color)) {
		return false;
	}

	return dark;
}
```

Also, I see this come back again and again. Care to pick this up in a follow-up PR and put this in a util/helper? Thanks!

Reply:
Yes, I will make a follow-up commit for this.

 :ok: by @Kemosabert on 28/06/2018 10:46:13